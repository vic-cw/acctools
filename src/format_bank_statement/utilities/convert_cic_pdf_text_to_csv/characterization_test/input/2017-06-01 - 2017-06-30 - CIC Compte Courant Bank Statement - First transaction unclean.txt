                                                                       Crédit Industriel et Commercial

                                                                                                                 CIC PARIS SAINT MARTIN

                                                   ¤                                                    224 RUE SAINT MARTIN 75010 PARIS
                                                           u0820 81 23 45 (Service 0,12 /min + prix appel) FAX 01 44 16 81 39
                                                                                                                                            12345@cic.fr                                        BIC : CMCIFRPP

                                                                                                                                                                                                                45678

                                                                                                                                                                                                                12345

                                                                                                                           M  BOB  BROWN

                                                                                                                           1  RUE   DE   REUILLY

                                                                                                                           75001    NEW YORK

...........................................................................................................................V..O...T.R..E...C..O..N..S..E..I.L..L..E..R..:..M...S..é..b..a.s..t.i.e.n...B..I..D..U..L.E......................

                                             RELEVE ET INFORMATIONS BANCAIRES                                                                                                                                Agence 12345

                                             30 juin 2017

                                                   C/C CONTRAT PERSONNEL GLOBAL N° 00012345678 en euros (GD)

                                                   TITULAIRE(S) : M BOB BROWN

                                             Date        Date valeur                    Opération                                                                                               Débit EUROS  Crédit EUROS

                                                                                        SOLDE CREDITEUR AU 31/05/2017                                                                                        1.234,56                                                                                        Pour tout appel en vue d'obtenir la bonne exécution d'un contrat ou le traitement

                                             01/06/2017    01/06/2017                  ONE LINE EXPENSE                                                                                         123,45                                        d'une réclamation, appelez le 09 69 32 06 06 appel non surtaxé

KV.20170701.012345.6789.1011.1213 X 0 2      05/06/2017    05/06/2017                  TWO LINE EXPENSE                                                                                         50,00

                                                                                       PLACE CARTE 00123456


                                             (GE) : protégé par la Garantie de l'Etat                                                                                                                        <<Suite au verso>>

                                             (GD) : protégé par la Garantie des Dépôts

                                             Remarque : Le livret réglementé relève de la garantie d'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s) de la garantie des dépôts.

                                             www.garantiedesdepots.fr

                                                                                                                                                                                                             Page 1

                                               CREDIT INDUSTRIEL ET COMMERCIAL (CIC) - Banque régie par les articles L.511-1 et suivants du Code Monétaire et Financier - SA au capital de 608 439 888 

                                         6,  avenue de Provence 75009 Paris - Adresse postale : 75452 Paris cedex 09 - tél 01 45 96 96 96 - swift CMCIFRPP - www.cic.fr - 542 016 381 RCS Paris - TVA intracommunautaire : FR34542016381

                                                                                        M le Médiateur du CIC : 63 chemin Antoine Pardon 69160 Tassin la demi lune - www.lemediateur-cic.fr

                                                                       Pour les opérations effectuées en sa qualité d'intermédiaire en opérations d'assurance : ORIAS 07 025 723 (www.orias.fr)
                                         RELEVE ET INFORMATIONS BANCAIRES                                                                                                                                  Agence 12345

                                         30 juin 2017

                                               C/C CONTRAT PERSONNEL GLOBAL N° 00012345678                            en  euros  (GD)

                                               TITULAIRE(S) : M BOB BROWN

                                         Date        Date valeur                    Opération                                                                                               Débit  EUROS   Crédit EUROS


                                         23/06/2017    23/06/2017                  REFUND                                                                                                                  2,46

                                                                                   PROUT

                                         (GE) : protégé par la Garantie de l'Etat

                                         (GD) : protégé par la Garantie des Dépôts

                                         Remarque : Le livret réglementé relève de la garantie d'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s) de la garantie des dépôts.

                                         www.garantiedesdepots.fr

                                                                                                                                                                                                           Page 2
                                                                       Crédit Industriel et Commercial

                                                                                                                 CIC PARIS SAINT MARTIN

                                                        ¤                                               224 RUE SAINT MARTIN 75010 PARIS
                                                           u0820 81 23 45 (Service 0,12 /min + prix appel) FAX 01 44 16 81 39
                                                                                                                                               12345@cic.fr                                            BIC : CMCIFRPP

                                             RELEVE ET INFORMATIONS BANCAIRES                                                                                                                                   Agence 12345

                                             30 juin 2017

                                                   C/C CONTRAT PERSONNEL GLOBAL N° 00012345678                                      en  euros  (GD)

                                                   TITULAIRE(S) : M BOB BROWN

                                             Date          Date valeur                  Opération                                                                                               Débit  EUROS    Crédit EUROS


                                             27/06/2017    27/06/2017                  PAIEMENT PSC 2506  NEUILLY                                                                                      2,94                                                                                               Pour tout appel en vue d'obtenir la bonne exécution d'un contrat ou le traitement

                                                                                       SELLER    CARTE 00123456

                                             Réf : 006                                  SOLDE CREDITEUR AU 30/06/2017                                                                                           0,00                 d'une réclamation, appelez le 09 69 32 06 06 appel non surtaxé

                                             QXBAN : AB01 CMCI FRPP XXX CDE2 FGHI 3JKL MNOP 4QR                                         IBAN : FR76 1234 5678 9101                                              1121 3141 516

                                               SITUATION DE VOS CREDITS en euros au 30 juin 2017

                                             Numéro                                     Libellé du compte                                      Montant emprunté                                                 Solde (*)

                                             00012345678               ALLURE LIBRE                                                     EUR                                                            +0,00         +0,00

                                             (*) Solde correspondant à l'amortissement normal du prêt

                                             AUTORISATIONS DE DECOUVERTS

KV.20170701.012345.6789.1011.1213 X 0 R      N° de compte                               Libellé                                                                                                                 Montant euros

                                             00012345678               DECOUVERT C. PERS. GLOBAL TAEG 15,12% AU 29/05/2017                                                                                             1.000.000,00

                                             TOTAL MENSUEL DES FRAIS DEBITES SUR PRODUITS ET SERVICES BANCAIRES

                                             N° de compte                               Libellé                                                                                                                 Montant euros

                                             00012345678               TOTAL DES FRAIS EN MAI 2017                                                                                                                     9,70

                                             (GE) : protégé par la Garantie de l'Etat

                                             (GD) : protégé par la Garantie des Dépôts

                                             Remarque : Le livret réglementé relève de la garantie d'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s) de la garantie des dépôts.

                                             www.garantiedesdepots.fr

                                                                                                                                                                                                                Page 3

                                               CREDIT INDUSTRIEL ET COMMERCIAL (CIC) - Banque régie par les articles L.511-1 et suivants du Code Monétaire et Financier - SA au capital de 608 439 888 

                                         6,  avenue de Provence 75009 Paris - Adresse postale : 75452 Paris cedex 09 - tél 01 45 96 96 96 - swift CMCIFRPP - www.cic.fr - 542 016 381 RCS Paris - TVA intracommunautaire : FR34542016381

                                                                                        M le Médiateur du CIC : 63 chemin Antoine Pardon 69160 Tassin la demi lune - www.lemediateur-cic.fr

                                                                       Pour les opérations effectuées en sa qualité d'intermédiaire en opérations d'assurance : ORIAS 07 025 723 (www.orias.fr)
