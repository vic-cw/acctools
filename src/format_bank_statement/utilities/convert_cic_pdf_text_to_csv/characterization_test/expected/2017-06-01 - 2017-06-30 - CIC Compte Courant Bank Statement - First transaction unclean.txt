2017-06-01,2017-06-01,"ONE LINE EXPENSE","-123,45"
2017-06-05,2017-06-05,"TWO LINE EXPENSE PLACE CARTE 00123456","-50,00"
2017-06-23,2017-06-23,"REFUND PROUT","2,46"
2017-06-27,2017-06-27,"PAIEMENT PSC 2506  NEUILLY SELLER    CARTE 00123456","-2,94"
